client/i18n/locales/english/intro.json

```diff
@@ -486,6 +486,10 @@
           "In the previous projects you learned how to work with basic DOM manipulation. Now it is time to review what you have learned by building a Rock, Paper, Scissors game.",
           "In this mini project, you will review conditionals, functions, <code>getElementById</code>, and more. This project will give you an opportunity to solve small problems and get a better understanding of the basics."
         ]
+      },
+      "learn-callbacks-by-building-a-testing-lib": {
```

Comment:
Also, I think this should be named library instead of lib because most campers won't understand what lib is